include/spatio_temporal_voxel_layer/measurement_buffer.hpp

```diff
@@ -137,6 +137,9 @@ class MeasurementBuffer
   // Removing old observations from buffer
   void RemoveStaleObservations(void);
 
+  rclcpp::Clock::SharedPtr clock_;
+  // Measurement buffer logger initialization simlular to observation buffer ?
```

Comment:
You don't need to initialize it here, I don't really like that we do that in nav2 :shrug: 

You can also remove that comment :-)

Reply:
Ok, I switch to an initialization in the constructor : `logger_(rclcpp::get_logger("measurement_buffer"))`
But not sure what the best practice here